File: worldcup/test/MatchTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class MatchTest {

    @Test
    void testInitMatch() {
        Match match = Match.initMatch("A", "B");

        assertEquals(0, match.getTeamOneScore());
        assertEquals(0, match.getTeamTwoScore());
    }
}

File: src/test/kotlin/MatchTest.kt
import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

internal class MatchTest {
    @Test
    fun updateScoreInitialScoreTest() {
        val game = getGameInstance()

        assertEquals(0, game.awayTeam.score)
        assertEquals(0, game.homeTeam.score)
    }

    @Test
    fun updateScoreSetStartScoreTest() {
        val game = getGameInstance()

        game.updateScore(1, 2)

        assertEquals(1, game.homeTeam.score)
        assertEquals(2, game.awayTeam.score)
    }

    fun getGameInstance() = Match(homeTeamName = "Chengdu United", awayTeamName = "Chelmsford United")
}